netcore/tools/jitdiff/jitdiff.cs

```diff
@@ -116,7 +117,7 @@ static bool TryParseFunctionName (string str, out string name)
 			// depends on objdump, let's use the whole line as a name if it ends with `:`
 			if (str.EndsWith (':'))
 			{
-				name = str;
+				name = Regex.Replace(str, @"(?i)\b([a-f0-9]+){8,16}\b", m => "0xD1FFAB1E");
```

Comment:
Maybe just replace with empty?

Reply:
Yeah we can, I just copied the coreclr's jitutills behavior (they replace all addresses with this const)